apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -110,7 +110,7 @@ export default {
       default: () => ({}),
     },
   },
-  emits: ["update:modelValue", "errorsInForm"],
```

Comment:
we don't need this anymore?

Reply:
Nope, I moved that functionality to the outer component in an earlier PR, but forgot to remove the emits here.